config/client.lua

```diff
@@ -0,0 +1,43 @@
+return {
+    UseTarget = GetConvar('UseTarget', 'false') == 'true',
```

Comment:
We don't have a convar for this so this can just be defaulted to false. Also, utilize camelCase for all of these

Reply:
ahh my bad, I was just following format of other resources but I guess they're not updated.